assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -47,15 +47,8 @@ export const TakeActionBoxoutEditor = ({
     imageUrl,
     imageAlt,
   } = useSelect(select => {
-    const args = {
-      per_page: -1,
-      sort_order: 'asc',
-      sort_column: 'post_title',
-      parent: window.p4_vars.take_action_page,
```

Comment:
Wrong place to get the variable. It is set `window.p4_vars.options.take_action_page`.
That's why `parent` was always `undefined` and no pages were retrieved.